repose-aggregator/tests/performance-tests/test_vars/filters/tenantculling.yml

```diff
@@ -0,0 +1,19 @@
+{
+  dir_prefix: "filters/tenantculling",
+  repose: {
+    config: {
+      system_model: {
+        filters: [
+          { name: "keystone-v2" },
+          { name: "tenant-culling" }
```

Comment:
I wish we could isolate this better.

Reply:
That would be nice, but we opted to reuse the data that the `keystone-v2` filter puts in the datastore instead of getting it all over again.